e2e/tests/node_not_ready/node_power_off.robot

```diff
@@ -0,0 +1,578 @@
+*** Settings ***
+Documentation       Test the Longhorn resillence if cluster node powering off
+
+Resource            ../../keywords/common.resource
+Resource            ../../keywords/engine.resource
+Resource            ../../keywords/replica.resource
+Resource            ../../keywords/node.resource
+Resource            ../../keywords/volume.resource
+
+Suite Setup         set_test_suite_environment
+Test Setup          set_test_environment    ${TEST NAME}
+Test Teardown       Cleanup resource and resume state
+
+
+*** Variables ***
+${Gi}=                  2**30
+${volume_size_gb}=      1
+${sleep_interval}=      300
+${volume_type}=         RWO
+
+
+*** Test Cases ***
+Node power off with replica-VA
+    [Documentation]    | =node 1= | =node 2= | =node 3= |
+    ...    | replica | replica | replica |
+    ...    | attached | | |
+    ...    | *power off* | | |
+    ${number_of_replicas}=    Convert To Integer    3
+    Set Test Variable    ${number_of_replicas}
+
+    ${volume_type}=    Evaluate    "${volume_type}".lower()
+
+    ${field1}=    Convert To String    {"spec": {"size": "${${volume_size_gb} * ${Gi}}"}}
+    ${field2}=    Convert To String    {"spec": {"numberOfReplicas": ${number_of_replicas}}}
+    ${field3}=    Convert To String    {"spec": {"accessMode": "${volume_type}"}}
+    @{list_of_fields}=    Create List    ${field1}    ${field2}    ${field3}
+
+    Given Create Volume With Fields    ${list_of_fields}
+    And Wait for all replicas to be created
+    And Get with replica node indices
```

Comment:
Any reason that you've moved this to the test cases?

Reply:
After @weizhe0422 explained, it is used in the latter steps (e.g `Replica state on node ${node_index} should eventually be ${expected_replica_state}`).

Suggest moving closer to the step where its used.

Reply:
After rethinking myself, I found that there is actually no need to have a specific step to fish out nodes with replicas, so I have removed this step and changed it to make judgments when checking the status of replicas.